.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,36 @@
+labels: ['bug']
+description: Create a report to help us improve
+body:
+ - type: markdown
+   attributes:
+      value: |
+        Thank you for coming here to submit a bug report. We care about anything from big to small relating to MHSF, making a better experience for you and others.
+
+        Try and be as descriptive as possible & type in a way thats able to understood by other contributors.
+ - type: textarea
+   attributes:
+     label: Current and Expected Behavior
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)